utils/misc.ml

```diff
@@ -549,13 +549,21 @@ let ordinal_suffix n =
 (* Color support handling *)
 module Color = struct
   external isatty : out_channel -> bool = "caml_sys_isatty"
+  external is_ansi_capable : out_channel -> bool = "caml_is_ansi_capable"
+  external set_ansi_capable : out_channel -> bool -> bool = "caml_set_ansi_capable"
```

Comment:
This should perhaps clarify that it returns `true` on sucesss (and not the old setting value).

Reply:
Fixed, thanks.